packages/x-telemetry/package.json

```diff
@@ -34,16 +34,18 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.26.10",
-    "@fingerprintjs/fingerprintjs": "^4.6.1",
+    "@fingerprintjs/fingerprintjs": "^3.4.2",
```

Comment:
Nice. 👍 
Could we add `renovate` rules to avoid bumping this and the `conf` packages over the wanted versions? 🤔